lib/src/sql/statements/info.rs

```diff
@@ -64,6 +64,20 @@ impl InfoStatement {
 					tmp.insert(v.name.to_string(), v.to_string().into());
 				}
 				res.insert("users".to_owned(), tmp.into());
+
+				// Process cluster membership
+				let mut memberships: Vec<Value> = vec![];
+				for cluster_member in run.scan_nd(0).await? {
+					let mut tmp = Object::default();
+					tmp.insert("nodeId".to_string(), Value::Strand(cluster_member.name.into()));
+					tmp.insert(
+						"heartbeat".to_string(),
```

Comment:
The heartbeats are determined with `scan_hb` :) You can use `tx.clock().await` to get the current time.

Reply:
Makes sense